foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_7_0/FoundationCoreJavaAiopsReferencesMigration.java

```diff
@@ -0,0 +1,93 @@
+package com.boozallen.aissemble.upgrade.migration.v1_7_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.AbstractAissembleMigration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.technologybrewery.baton.BatonException;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.boozallen.aissemble.upgrade.util.FileUtils.replaceLiteralInFile;
+
+public class FoundationCoreJavaAiopsReferencesMigration extends AbstractAissembleMigration {
+
+    public static final Logger logger = LoggerFactory.getLogger(FoundationCoreJavaAiopsReferencesMigration.class);
+    public static List<String> VERSION_AISSEMBLE_PRE_1_7_0 = new ArrayList<>();
+    public static List<String> VERSION_AISSEMBLE_POST_1_7_0 = new ArrayList<>();
+
+    /**
+     * Function to add packages to the VERSION_AISSEMBLE_PRE_1_7_0 & VERSION_AISSEMBLE_POST_1_7_0 list.
+     * @param n/a.
+     * @return n/a.
+     */
+    public static void addPackagesToList(){
```

Comment:
A: I think we might end up calling this multiple times.  Might just switch to using a static initialization that's common for these types of constant sets. Something like:
```
ublic static List<String> VERSION_AISSEMBLE_POST_1_7_0 = Collections.unmodifiableList(
   "com.boozallen.aips.core.cdi",
   ...,
   "com.boozallen.aiops.core.filestore"
)
```